pkgs/by-name/nu/numbat/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "numbat";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "sharkdp";
+    repo = "numbat";
+    rev = "v${version}";
+    hash = "sha256-r6uPe2NL+6r/fKjf0C/5DLdB5YP3SIo8g8EsDxKP/3g=";
+  };
+
+  cargoHash = "sha256-MPqJjCfIwgK8QigWQYfWAYlg9RNMzF4x+0SprS0raKY=";
+
```

Comment:
```suggestion

  buildInputs = lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.Security
  ];

```

Reply:
Why is  `darwin.apple_sdk.frameworks.Security` needed? Or why do I only need the `Security` part of the`apple_sdk`.

Reply:
Unfortunately I do not know where to find the document.
I just added it because the compiler reported that Security was missing.

Since apple_sdk.frameworks originally exists in the darwin environment, many packages implicitly depend on it, such as this package.
However, these are impure and need to be explicitly added to the input in nix.
The frameworks are listed in [this file](https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/apple-sdk/frameworks.nix), but it is too much to include them all, so only the necessary ones should be added.

Reply:
Yes I also found this file but it was a little overwhelming.
Thanks for clarification though.